ansible-scylla-node/tasks/main.yml

```diff
@@ -1,16 +1,24 @@
 ---
 
+# Facts gathering
 - name: Populate service facts
   service_facts:
 
 - name: Populate package facts
   package_facts:
     manager: auto
 
+# Upgrade
```

Comment:
so below will trigger update only when you flip the variable
then you will need to flip variable back so you will be able to be idempotent and not pickup new versions without knowing
OR is the upgrade version code idempotent?

Reply:
so it can be overriden on the command line just for this run then? (so we can keep config files in git without adjusting their node_params.yml ? )
(try to think about the jenkins approach we use, how would it fit there and the git config store)

Reply:
You can use the command line to load the playbook instead. But, as far as I know, everything you define at CLI level has top precedence.